src/main/java/nextstep/subway/path/domain/Fare.java

```diff
@@ -0,0 +1,43 @@
+package nextstep.subway.path.domain;
+
+import nextstep.subway.line.domain.Section;
+
+import java.util.List;
+
+public class Fare {
+    private final static Long DEFAULT_FARE = 1250L;
+    private final Long fare;
```

Comment:
```suggestion
    private static final Long DEFAULT_FARE = 1250L;
    
    private final Long fare;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가한다면 가독성이 상승합니다 ~
또한 final 보다 static을 먼저 작성해주세요!

https://stackoverflow.com/questions/16731240/what-is-a-reasonable-order-of-java-modifiers-abstract-final-public-static-e